src/ncp/ncp_base.cpp

```diff
@@ -5689,14 +5668,13 @@ otError NcpBase::SetPropertyHandler_MAC_WHITELIST(uint8_t aHeader, spinel_prop_k
         }
 
         VerifyOrExit(parsedLength > 0, error = OT_ERROR_PARSE);
-
-        if (rssi == RSSI_OVERRIDE_DISABLED)
+        if (rssi == OT_RSSI_OVERRIDE_DISABLED)
         {
-            SuccessOrExit(error = otLinkAddWhitelist(mInstance, extAddress->m8));
+            error = otLinkAddWhitelist(mInstance, extAddress->m8);
```

Comment:
Updated. will continue to add if OT_ERROR_NONE or OT_ERROR_ALREADY, and exit otherwise.